airflow/utils/log/wasb_task_handler.py

```diff
@@ -1,175 +1,177 @@
-
-        if self.wasb_log_exists(remote_loc):
-            # If Wasb remote file exists, we do not fetch logs from task instance
-            # local machine even if there are errors reading remote logs, as
-            # returned remote_log will contain error messages.
-            remote_log = self.wasb_read(remote_loc, return_error=True)
-            log = '*** Reading remote log from {}.\n{}\n'.format(
-                remote_loc, remote_log)
-        else:
-            log = super(WasbTaskHandler, self)._read(ti, try_number)
-
-        return log
-
-    def wasb_log_exists(self, remote_log_location):
-        """
-        Check if remote_log_location exists in remote storage
-        :param remote_log_location: log's location in remote storage
-        :return: True if location exists else False
-        """
-        try:
-            return self.hook.check_for_blob(self.wasb_container, remote_log_location)
-        except Exception:
-            pass
-        return False
-
-    def wasb_read(self, remote_log_location, return_error=False):
-        """
-        Returns the log found at the remote_log_location. Returns '' if no
-        logs are found or there is an error.
-        :param remote_log_location: the log's location in remote storage
-        :type remote_log_location: string (path)
-        :param return_error: if True, returns a string error message if an
-            error occurs. Otherwise returns '' when an error occurs.
-        :type return_error: bool
-        """
-        try:
-            return self.hook.read_file(self.wasb_container, remote_log_location)
-        except AzureHttpError:
-            msg = 'Could not read logs from {}'.format(remote_log_location)
-            self.log.exception(msg)
-            # return error if needed
-            if return_error:
-                return msg
-
-    def wasb_write(self, log, remote_log_location, append=True):
-        """
-        Writes the log to the remote_log_location. Fails silently if no hook
-        was created.
-        :param log: the log to write to the remote_log_location
-        :type log: string
-        :param remote_log_location: the log's location in remote storage
-        :type remote_log_location: string (path)
-        :param append: if False, any existing log file is overwritten. If True,
-            the new log is appended to any existing logs.
-        :type append: bool
-        """
-        if append and self.wasb_log_exists(remote_log_location):
-            old_log = self.wasb_read(remote_log_location)
-            log = '\n'.join([old_log, log]) if old_log else log
-
-        try:
-            self.hook.load_string(
-                log,
-                self.wasb_container,
-                remote_log_location,
-            )
-        except AzureHttpError:
-            self.log.exception('Could not write logs to %s',
-                               remote_log_location)
+# -*- coding: utf-8 -*-
```

Comment:
This is line-endings I assume.

Reply:
Your are perfectly correct, the file was using DOS style line-endings and now unifying then to use UNIX style, hope this is fine.